subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv.c

```diff
@@ -1167,6 +1155,7 @@ static void isr_done(void *param)
 	extra->type = EVENT_DONE_EXTRA_TYPE_ADV;
 #endif  /* CONFIG_BT_CTLR_ADV_EXT */
 
+	HDR_RESULT_SET(lll, DONE_COMPLETED);
```

Comment:
```suggestion
```

As `DONE_COMPLETED` can be assigned to `0` in the enum, then an explicit set of this reason can be avoided.

Reply:
So reset it to `0` in the ull_hdr_init, or how do you make sure it's default `0`?